PSix/Array.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Array.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/29/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+
+extension Array {
+    
+    static func filterNils(array: [T?]) -> [T] {
```

Comment:
After wasting an hour debugging cocoa pods for this, I am opting out. I definitely see the advantage of Dollar, but I have no idea how to integrate it.